lib/clearance/password_strategies/bcrypt_migration_from_sha1.rb

```diff
@@ -1,6 +1,13 @@
 module Clearance
   module PasswordStrategies
     module BCryptMigrationFromSHA1
+      DEPRECATION_MESSAGE = "[DEPRECATION] The BCryptMigrationFromSha1 " \
```

Comment:
If I'm not mistaken, the choice is between a heredoc and proper indentation? I'd rather have the indentation. Any other options?

If I use a heredoc, I think I have to leaves the messages left aligned in the source or I'll actually get those spaces in the terminal output. Is that correct?